docs/user/README.md

```diff
@@ -86,28 +114,121 @@ AZURE_STORAGE_ACCOUNT_NAME
 AZURE_STORAGE_ACCESS_KEY
 ```
 
-Assuming a container named `wind-turbine` already exists, `modelardbd` can then be run in edge mode with transfer of
-the ingested time series to the [Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/)
-container `wind-turbine`:
+Assuming a container named `wind-turbine` already exists, `modelardbm` can then be started with transfer of the ingested 
+time series to the [Azure Blob Storage](https://azure.microsoft.com/en-us/products/storage/blobs/) container `wind-turbine`:
 
 ```shell
-modelardbd edge path_to_local_data_folder azureblobstorage://wind-turbine
+modelardbm metadata azureblobstorage://wind-turbine
 ```
 
-To run `modelardbd` in cloud mode simply replace `edge` with `cloud` as shown below. Be aware that both a local data
-folder and an object store are required when `modelardbd` is run in cloud mode.
+When a manager is running, `modelardbd` can be started in edge mode with transfer of the ingested time series to the
+object store given to `modelardbm` using the following command:
 
 ```shell
-modelardbd cloud path_to_local_data_folder s3://bucket-name
+modelardbd edge path_to_local_data_folder manager_url
+````
+
+To run `modelardbd` in cloud mode simply replace `edge` with `cloud` as shown below. Be aware that when running in cloud
+mode the `modelardbd` instance will execute queries against the object store given to `modelardbm` and not against local
+storage.
+
+```shell
+modelardbd cloud path_to_local_data_folder manager_url
 ```
 
-Note that the server uses `9999` as the default port. The port can be changed by specifying a different port with an
-environment variable:
+In both cases, access to the PostgreSQL metadata database and the object store is handled by the manager and is set up
```

Comment:
Maybe write it ".... object store is automatically provided by the manager when the `modelardbd` is connected to it.

Reply:
Fixed.